test/e2e/multiple_namespaces_test.go

```diff
@@ -0,0 +1,129 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/vmware-tanzu/velero/pkg/util/kube"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/pkg/errors"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+var _ = Describe("[Basic] Backup/restore of 2 namespaces", func() {
+	var (
+		client *kubernetes.Clientset
+	)
+
+	BeforeEach(func() {
+		var err error
```

Comment:
The `uuidgen` variable is a global across the e2e package. We should be calling `uuidgen, err = uuid.NewRandom()` somewhere here as it is not initialised within this file unlike other tests. It means that if you run only the scale test, it isn't set and so all the characters of the UUID are `0`.